ui/pb/progressbar.go

```diff
@@ -32,29 +32,47 @@ import (
 const defaultWidth = 40
 const defaultBarColor = color.Faint
 
-// ProgressBar is just a simple thread-safe progressbar implementation with
+// Status of the progress bar
+type Status string
+
+// Progress bar status symbols
+const (
+	Running     Status = " "
+	Waiting     Status = "•"
+	Stopping    Status = "↓"
+	Interrupted Status = "✗"
+	Done        Status = "✓"
+)
+
+var statusColors = map[Status]*color.Color{
```

Comment:
This was intentional, and I still feel a global var is the right choice in this case. Should I disable the linter here?

Reply:
I think it's fine. I find it somewhat strange that you only color two of the statuses ... although I don't know if we should color the others either way so :D

Reply:
Yeah, I don't think the other statuses need color, but we can consider it. :)

I'm ignoring this warning now, but I'm starting to see the light with some of the decisions made for Rust... Go should at least have built-in support for immutable objects, since making all immutable by default will probably never happen. It would help in cases like this where implementing it otherwise would be awkward.

Reply:
Yeah, it's perfectly ok to ignore the linter warnings for maps. The other way to do something like that without linter errors is to wrap it in a function, which doesn't make any sense, so `//nolint:gochecknoglobals` is the way to go unfortunately.